examples/OktoResources/ResourceRelations/Connection/06.external-target-prometheus-connection-tls-config.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: blocks/v1beta1
+kind: Connection
+metadata:
+  name: connection-with-external-target
+  namespace: default
+  labels:
+    site.nbycomp.com/{{ .Values.placement.site.label }}: "true"
+    kind: K8sCluster
+spec:
+  externalTarget:
+    prometheus:
+      query: |-
+        (avg by (instance) 
+          (rate(node_cpu_seconds_total{job="node",mode="idle"}[1m])) * 100)
+        > 70
+      endpoint:
+        insecureValue: |
+          address: http://prometheus:9090
+          serverCertificateAuthority: | {{ .Values.prometheus.endpoint.serverCertificateAuthority | nindent 12 }}
+          clientKeyPair:
+            certificate: | {{ .Values.prometheus.endpoint.clientCertificate | nindent 14 }}
+            key: | {{ .Values.prometheus.endpoint.clientKey | nindent 14 }}
```

Comment:
Let's make an example for a prometheus config with this. Let's not use {{ .Values... }} there, and use throwaway certs.

Reply:
To clear up ambiguity: Let's make a prometheus config typed connection instead. The external target doesn't get directly affected by this.